queries.sql

```diff
@@ -1,3 +1,70 @@
-/*Queries that provide answers to the questions from all projects.*/
+/* Queries that provide answers to the questions from all projects. */
+/* Select elements that contains mon */
+SELECT
+  *
+FROM
+  animals
+WHERE
+  name LIKE '%mon%';
 
-SELECT * from animals WHERE name = 'Luna';
+/* Select elements between 2016 and 2019 */
+SELECT
+  *
+FROM
+  animals
+WHERE
+  date_of_birth BETWEEN '2016-01-01'
+  AND '2019-01-01';
+
```

Comment:
- [ ] This query is good. But what if an animal was born on the 21st March 2019? Currently, your query doesn't cover those animals. You only cover animals born on the 1st day and 1st month of 2016 and 2019.
Please update this query to cover animals born in between the given years.

```sql
`SELECT name FROM animals WHERE EXTRACT(year FROM date_of_birth) BETWEEN 2016 AND 2019;`
```

Reply:
hi, @mmsesay  thanks a lot for your review I didn't think about that case 😵‍💫